src/styles/components/navbar.sass

```diff
@@ -82,11 +82,11 @@ $wp-max-width: 940px
   padding: 0.5rem 0.75rem
   position: relative
   .icon
-    margin-left: $space-small
+    margin-inline-start: $space-small
 
 .navbar.embedded
   .navbar-end
-    margin-left: auto
+    margin-inline-start: auto
```

Comment:
I haven't even thought about converting the left-right values to inline-start and inline-end! Cool!

Reply:
There are still 4 occurrences of `-left` in this file, and one of them is causing the navbar burger to not be `inline-end`-aligned on mobile view in RTL. So, on RTL page the burger is next to Openverse logo.

Reply:
I'm not able to reproduce this, the burger is positioned correctly for me:
<img width="291" alt="Screenshot 2021-11-25 at 3 21 20 PM" src="https://user-images.githubusercontent.com/16580576/143432547-d64fcd42-11a6-4ce8-99e0-3b83a46c2ecd.png">
<img width="291" alt="Screenshot 2021-11-25 at 3 22 05 PM" src="https://user-images.githubusercontent.com/16580576/143432669-f45550df-273d-46e4-b616-7dee47fafe81.png">

Reply:
I re-checked out the branch, and everything looks great, thank you!